tarodb/sqlc/migrations/000002_assets.up.sql

```diff
@@ -31,6 +31,21 @@ CREATE TABLE IF NOT EXISTS genesis_points (
     anchor_tx_id INTEGER REFERENCES chain_txns(txn_id)
 );
 
+-- assets_meta is a table that holds all the metadata information for genesis
+-- assets that we either created, or bootstrapped from the relevant Base
+-- Universe.
+CREATE TABLE IF NOT EXISTS assets_meta (
+    meta_id INTEGER PRIMARY KEY,
+
+    meta_data_hash BLOB CHECK(length(meta_data_hash) = 32) UNIQUE,
+
+    -- TODO(roasbeef): also have other opque blob here for future fields?
+    meta_data_blob BLOB,
+
+    meta_data_type SMALLINT
+);
+CREATE INDEX IF NOT EXISTS meta_hash_index on assets_meta(meta_data_hash);
```

Comment:
Thanks to the `UNIQUE` keyword I think we don't need an explicit index: https://stackoverflow.com/questions/3379292/is-an-index-needed-for-a-primary-key-in-sqlite